### PR TITLE
Slider: feature - add format-tooltip prop

### PR DESCRIPTION
**Pull request by liyanlong**
* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

- 给Slider 提供 格式化tooltip content 的功能
- 补充了中文英文 Slider 文档
- 提供了单元测试

参考 #3430

### CODE REVIEW

File: examples/docs/en-US/slider.md [hunk 1]
```diff
@@ -166,6 +182,7 @@ Selecting a range of values is supported.
 | show-input-controls | whether to display control buttons when `show-input` is true | boolean | — | true |
 | show-stops | whether to display breakpoints | boolean | — | false |
 | show-tooltip | whether to display tooltip value | boolean | — | true |
+| format-tooltip | format to display tooltip value | function | — | — |
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/slider.md]: 最好写一下 `format-tooltip` 的参数吧，在 `Description` 一列：
a function to format the displayed tooltip value, and the value will be passed as its parameter

File: examples/docs/zh-CN/slider.md [hunk 2]
```diff
@@ -190,6 +206,7 @@
 | show-input-controls | 在显示输入框的情况下，是否显示输入框的控制按钮 | boolean | — | true|
 | show-stops | 是否显示间断点 | boolean | — | false |
 | show-tooltip | 是否显示 tooltip | boolean | — | true |
+| format-tooltip | 格式化 tooltip message | function | — | — |
```
Comment (Leopoldthecoder) [inline examples/docs/zh-CN/slider.md]: 同上。

File: examples/docs/en-US/slider.md (hunk 1 above)
Comment (liyanlong) [inline examples/docs/en-US/slider.md]: 恩， 已经改成 ```Function(value)```

File: examples/docs/zh-CN/slider.md (hunk 2 above)
Comment (liyanlong) [inline examples/docs/zh-CN/slider.md]: 同上，已改

**Comment by liyanlong** [comment]
@Leopoldthecoder  resolved

**Pull request merged**